### PR TITLE
Feature/86cwe1w8q: Deployment

### PR DESCRIPTION
**Pull request by joshualoongwy**
https://app.clickup.com/t/86cwe1w8q

Added a dockerfile which creates a production bundle for our app.

To build this docker image, go to behind-the-veil-siteroot and run:

`docker build -t behind-the-veil .`

Note: If you're on a mac with an ARM chip, instead, run:
`docker buildx build -t behind-the-veil --platform linux/amd64 .`

Then once it's built, to run the app in a container locally, run:
`docker run -e PORT=8080 -e MONGO_URL='<mongo_url>' -e MAIL_URL='<mail_url>' -e FROM_USER='<email_details>' -p 8080:8080 behind-the-veil`

But replace the env vars with our secrets.

I'll include more details on how to actually deploy using GCP in our readme, but here are my rambling notes for now: https://docs.google.com/document/d/1HinuPDp9W3cqWvwL_1eiB8-TITT-PstmT1eVcHspUqw/edit

### CODE REVIEW

**Review by nethbtju — APPROVED** [review]
No issues, seems good to go!

File: behind-the-veil-siteroot/server/main.js [hunk 1]
```diff
@@ -45,7 +45,9 @@ Meteor.startup(async () => {
 
     if (Meteor.isServer) {
         // process.env.MAIL_URL = mailUrl;
-        Accounts.emailTemplates.from = fromUser;
+        // Accounts.emailTemplates.from = fromUser;
```
Comment (nethbtju) [inline behind-the-veil-siteroot/server/main.js:48]: do you reckon we need this anymore then?

Comment (joshualoongwy) [reply behind-the-veil-siteroot/server/main.js:48]: nah good point ill get rid of it

**Review by nethbtju — APPROVED** [review]
Good to go

**Pull request merged**